### ISSUE TITLE
KeyError: 'library/enum'

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1343]
The following happened when building the latest cpython tip documentation. I don't think it's a bug in Sphinx, but the error message told me to report it, so this is what I'm doing.

```
#!sh

 $ make html
mkdir -p build/html build/doctrees
python tools/sphinx-build.py -b html -d build/doctrees -D latex_paper_size=  . build/html 
Running Sphinx v1.0.7
loading pickled environment... done
building [html]: targets for 437 source files that are out of date
updating environment: [extensions changed] 434 added, 0 changed, 3 removed
reading sources... [100&#37;] whatsnew/index                                                                       
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [ 42&#37;] library/enum                                                                          
Exception occurred:
  File "/home/nikratio/clones/cpython/Doc/tools/sphinxext/pyspecific.py", line 46, in new_visit_literal_block
    meta = self.builder.env.metadata[self.builder.current_docname]
KeyError: 'library/enum'
The full traceback has been saved in /tmp/sphinx-err-kQuGXN.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
Either send bugs to the mailing list at <http://groups.google.com/group/sphinx-dev/>,
or report them in the tracker at <https://bitbucket.org/birkenfeld/sphinx/issues/>. Thanks!
make: *** [build] Error 1
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1343
- Originally reported by: Nikolaus Rath
- Originally created at: 2014-01-12T03:17:23.548

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-19 16:31:32+00:00_

Should be fixed meanwhile since we updated to 1.2.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-10-18 07:32:18+00:00_

Removing version: 1.0.7 (automated comment)
